docker/download-mutinynet.sh

```diff
@@ -59,17 +59,23 @@ if [[ ${fedimintd_install[*]} =~ ^[Yy]?$ ]]; then
   echo "Running 'docker-compose -f $FEDIMINTD_DIR/docker-compose.yaml up -d' to start fedimintd"
   docker-compose -f $FEDIMINTD_DIR/docker-compose.yaml up -d
   echo "Optionally run 'docker-compose -f $FEDIMINTD_DIR/docker-compose.yaml logs -f' to see the logs"
-  echo "You can access the fedimintd dashboard at http://$external_ip:3000"
-  echo "Note: by default you should open ports 8173 and 8174 for external access on your router/firewall, plus the ports mentioned above"
   echo
 fi
 
 if [[ ${gateway_install[*]} =~ ^[Yy]?$ ]]; then
   echo "Running 'docker-compose -f $GATEWAY_DIR/docker-compose.yaml up -d' to start the LN gateway"
   docker-compose -f $GATEWAY_DIR/docker-compose.yaml up -d
   echo "Optionally run 'docker-compose -f $GATEWAY_DIR/docker-compose.yaml logs -f' to see the logs"
+  echo
+fi
+
+if [[ ${fedimintd_install[*]} =~ ^[Yy]?$ ]]; then
+  echo "You can access the fedimintd dashboard at http://$external_ip:3000"
+  echo "Note: by default you should open ports 8173 and 8174 for external access on your router/firewall, plus the ports mentioned above"
+fi
+
+if [[ ${gateway_install[*]} =~ ^[Yy]?$ ]]; then
```

Comment:
Since this variable is compared repeatedly, maybe we should set a variable that's easier to test, although the lack of support for boolean variable in bash could reduce the benefit.